tests/providers/gcp/gcp_test.py

```diff
@@ -581,6 +583,20 @@ def testListDisksByLabels(self, mock_gce_api, mock_labels):
       disk_names = []
     self.assertEqual(0, len(disk_names))
 
+  @typing.no_type_check
+  @mock.patch('libcloudforensics.providers.gcp.internal.common.GoogleCloudComputeClient.BlockOperation')
+  @mock.patch('libcloudforensics.providers.gcp.internal.compute.GoogleComputeImage')
+  @mock.patch('libcloudforensics.providers.gcp.internal.common.GoogleCloudComputeClient.GceApi')
+  def testCreateImageFromGcsTarGz(self, mock_gce_api, mock_gce_image, mock_block_operation):
+    """Test that images are correctly imported from compressed tar archives in GCS."""
+    mock_block_operation.return_value = None
+    mock_gce_image.return_value = FAKE_IMAGE
+    image_insert = mock_gce_api.return_value.images.return_value.insert
+    image_insert.return_value.execute.return_value = {'name': 'fake-image'}
+    image_object = FAKE_ANALYSIS_PROJECT.compute.CreateImageFromGcsTarGz(
+        'gs://fake-bucket/fake-folder/image.tar.gz', 'fake-image')
+    self.assertIn('fake-image', image_object.name)
```

Comment:
It would also be good to assert that `gce_image_client.insert` is called with the parameters you're expecting. You can use `image_insert.assert_called_with` for that

Reply:
done